doc/api/packages.md

```diff
@@ -364,12 +364,13 @@ For example, a package that wants to provide different ES module exports for
 Node.js supports the following conditions out of the box:
 
 * `"import"` - matched when the package is loaded via `import` or
-   `import()`. Can reference either an ES module or CommonJS file, as both
-   `import` and `import()` can load either ES module or CommonJS sources.
-   _Always matched when the `"require"` condition is not matched._
+   `import()`, or via any top-level load or resolve operation by the
```

Comment:
> or via any top-level load or resolve operation by the ECMAScript module loader.

are there any other such operations?

Reply:
Only for tools like bundlers etc currently as you cannot set a package as a top-level entry into Node.js. But eg if we were to permit `new Worker('import:external-dep')` or something it should follow the same logic. Not that we do but trying to capture the overall spirit of it.

Reply:
I updated the wording here to use "top-level import" over "top-level load". Hopefully that clarifies a bit better.